demo_modules/slither/slither.js

```diff
@@ -0,0 +1,210 @@
+const register = require('register');
+const ModuleInterface = require('lib/module_interface');
+const _ = require('underscore');
+const debug = require('debug');
+const state = require('state');
+const wallGeometry = require('wallGeometry');
+
+const COLORS = [
+  '#ff0000',
+  '#00ff00',
+  '#0000ff',
+  '#ff00ff',
+  '#cdc9c9',
+  '#ffdead',
+  '#bebebe',
+  '#e6e6fa',
+  '#8470ff',
+  '#00bfff',
+  '#40e0d0',
+  '#2e8b57',
+  '#32cd32',
+  '#ffff00',
+  '#ffd700',
+  '#cd5c5c',
+  '#b22222',
+  '#f4a460',
+  '#ffa500',
+  '#ff6347',
+  '#d02090',
+  '#8a2be2',
+]
+const SPEED = 0.2;
+
+const isOutOfBounds = ({x, y}, rect) => {
+  return x < rect.x || y < rect.y || x > rect.x + rect.w || y > rect.y + rect.h;
+};
+
+const darken = (color, i) => {
```

Comment:
is it worth using a real color object and only doing the conversion to strings for color fewer times ? I assume this runs fine as-is, so maybe just a TODO would be nice.

Reply:
Done.